README.md

```diff
@@ -76,16 +74,17 @@ If you use vanilla WordPress:
 
 - Comment out the above line to disable profiling.
 
-If you want to stop profiling, you can just comment/remove those lines.
+To stop profiling, comment/remove those lines.
 
-Take into account that with the default configuration, every time you `ddev start`, DDEV will recreate this file. You can remove the `#ddev-generated` at the top of the file if you want to avoid that.
+Take into account that with the default configuration, every time you `ddev start`, DDEV will recreate this file. Remove the `#ddev-generated` at the top of the file to avoid that.
```

Comment:
It's better to just tell them what to do in this situation to begin with, remove #ddev-generated and then make the edits. 

The reality is we need to make that file include overrides by default. (Which is a whole different conversation.)

Reply:
Fixed in e62ff29d30aa1bad0029b7eb985973fa6e82e695